_rules/SC1-1-1-accessible-name-is-image-filename.md

```diff
@@ -0,0 +1,117 @@
+---
+name: Filename is valid accessible name
+description: |
+  This rule checks that image elements that use their source filename as their accessible name do so without loss of information to the user.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML `input` element with a [`type`](https://www.w3.org/TR/html/sec-forms.html#dom-htmlinputelement-type) of `image`, or any HTML element with the [semantic role](#semantic-role) of `img`, is [included in the accessibility tree](#included-in-the-accessibility-tree), and has an [accessible name](#accessible-name) that is equivalent to the [filename](#filename) specified in the `src` attribute. Difference in letter casing, and forward and trailing whitespace should be ignored.
```

Comment:
editorial: is [included in the accessibility tree] > which is [included in the accessibility tree]

Isn't the last sentence an assumption?